lms/static/my.css

```diff
@@ -1,3 +1,18 @@
+@media screen and (max-width: 30em) {
```

Comment:
Specifically here, I would probably use `px`. (~768px)

Reply:
Can you explain why so big viewport to start removing items from the view?

Reply:
Ignore my question. I modified the code accordingly.